lua/ogpt/actions.json

```diff
@@ -49,7 +50,7 @@
   },
   "optimize_code": {
     "type": "popup",
-    "strategy": "edit",
+    "strategy": "display",
     "template": "Optimize the following code.\n\nCode:\n```{{{filetype}}}\n{{{input}}}\n```\n\nOptimized version:\n```{{{filetype}}}"
```

Comment:
Generally speaking, I'm quite confused about the syntax of "template". Should the arguments be wrapped with triple backticks ```? None of the arguments in this line is properly wrapped by triple backticks. Arguments in line 10 are not wrapped at all.